clang/include/clang/Driver/Options.td

```diff
@@ -6976,6 +6979,9 @@ let Visibility = [CC1Option, CC1AsOption] in {
 def debug_info_macro : Flag<["-"], "debug-info-macro">,
   HelpText<"Emit macro debug information">,
   MarshallingInfoFlag<CodeGenOpts<"MacroDebugInfo">>;
+def no_system_debug : Flag<["-"], "no-system-debug">,
+  HelpText<"Do not emit debug information for declarations in system headers">,
+  MarshallingInfoFlag<CodeGenOpts<"NoSystemDebug">>;
```

Comment:
Instead of having `-fno-system-debug` and `-no-system-debug` we can probably just have `-fno-system-debug` to handle both the Driver and -cc1 command lines.

Reply:
I based my option handling on (-fdebug-macro/-fno-debug-macro) which controls generation of the cc1 option -debug-info-macro option.   Which way is preferred?

>   // -fdebug-macro turns on macro debug info generation.
>   if (Args.hasFlag(options::OPT_fdebug_macro, options::OPT_fno_debug_macro,
>                    false))
>     if (checkDebugInfoOption(Args.getLastArg(options::OPT_fdebug_macro), Args,
>                              D, TC))
>       CmdArgs.push_back("-debug-info-macro");

Reply:
I would prefer using `-fno-system-debug` across the board.  It also simplifies the driver implementation, as then all you need to do is call `A->render(Args, CmdArgs)` when we know the option is passed on the command line.